easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-v1706-foss-2016b.eb

```diff
@@ -0,0 +1,44 @@
+# -*- python -*-
+name = 'OpenFOAM'
+version = 'v1706'
+
+homepage = 'http://www.openfoam.com/'
+description = """OpenFOAM is a GPL-opensource C++ CFD-toolbox.
+This offering is not approved or endorsed by OpenCFD Ltd,
+producer and distributor of the OpenFOAM software via www.openfoam.com,
+and owner of the OPENFOAM trademark.
+"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = [
+    ('https://sourceforge.net/projects/openfoamplus/files/%(version)s',
+     'download')
+]
+
+sources = [ 'OpenFOAM-%(version)s.tgz' ]
+checksums = ['630d30770f7b54d6809efbf94b7d7c8f']
+
+patches = [ 'OpenFOAM-%(version)s-cleanup.patch' ]
+
+dependencies = [
+    # > causes bash issue? ('libreadline', '6.3'),
+    # > causes bash issue? ('ncurses', '6.0'),
+    # OpenFOAM requires 64 bit METIS using 32 bit indexes (array indexes)
+    ('METIS', '5.1.0'),
+    ('SCOTCH', '6.0.4'),
+    ('Boost', '1.61.0'),
+    # Could build with CGAL, but this adds an extra python dependency
+    # that we don't want/need
+    # ('CGAL', '1.60.0'),
+    # ('ParaView', '5.4.0', '-mpi'),
```

Comment:
If I just want OpenFOAM, why a huge slew of python things? With paraview it would likely be good to distinguish between an off-screen and hardware mode too. However, I didn't want to get into this. All I wanted was to get a new version of OpenFOAM building with easybuild.